lib/cocoapods-playgrounds/workspace.rb

```diff
@@ -77,7 +78,11 @@ def target_name
 
     def workspace_path
       extension = @tool == :cocoapods ? 'xcworkspace' : 'xcodeproj'
-      target_dir + "#{names.first}.#{extension}"
+      if @use_legacy_xcode
```

Comment:
This change was based on (https://github.com/segiddins/ThisCouldBeUsButYouPlaying/pull/44). I thought I might need it to get it working.